pkgs/os-specific/linux/nvidia-x11/builder.sh

```diff
@@ -191,8 +191,8 @@ installPhase() {
                     --set-rpath $out/lib:$libPath $bin/bin/$i
             fi
         done
-        # FIXME: needs PATH and other fixes
-        # install -Dm755 nvidia-bug-report.sh $bin/bin/nvidia-bug-report.sh
+        sed -E 's#/bin/grep#grep#' nvidia-bug-report.sh > nvidia-bug-report.sh.fixed
+        install -Dm755 nvidia-bug-report.sh.fixed $bin/bin/nvidia-bug-report.sh
```

Comment:
```suggestion
        substituteInPlace nvidia-bug-report.sh --replace /bin/grep ${grep}/bin/grep
        install -Dm755 nvidia-bug-report.sh $bin/bin/nvidia-bug-report.sh
```

Reply:
Switched to use substituteInPlace. I recommend keeping the grep invocation path-relative seeing as the rest of the grep calls in the script are relative. The alternative is declaring gnugrep as an explicit dependency in the derivation, which seems unnecessary, especially as there's plenty of other unqualified calls to posix tools in the script.

Reply:
I agree that fixing all the relative calls could be a pain, but perhaps we could use `wrapProgram` and append the expected tools to `PATH` to ensure the dependencies are actually available at runtime?

Reply:
We also can use `resholve.mkDerivation` for those scripts. It is kinda of a pain to get it right, but it has the advantage of ensuring that no undeclared dependencies is used.